packages/auth/src/grant/routes.ts

```diff
@@ -115,7 +120,7 @@ async function createGrant(
     throw new GNAPServerRouteError(
       400,
       GNAPErrorCode.InvalidRequest,
-      'access identifier required'
+      err instanceof Error ? err.message : ''
```

Comment:
to avoid empty string, let's provide some generic message instead at least. It's less important for the client, its more so for us to find where to debug it.